### PR TITLE
fluent: docs: update grep for style

### PR DESCRIPTION
**Pull request by esmerel**

### CODE REVIEW

File: pipeline/filters/grep.md [hunk 1]
```diff
@@ -1,28 +1,33 @@
...
+| `Regex`      | KEY REGEX | Keep records where the content of KEY matches the regular expression. |
+| `Exclude`    | KEY REGEX | Exclude records where the content of KEY matches the regular expression. |
+| `Logical_Op` | Operation | Specify a logical operator:  `AND`, `OR` or `legacy` (default). In `legacy` mode the behaviour is either `AND` or `OR` depending on whether the `grep` is including (uses AND) or excluding (uses OR). Available from 2.1 or higher. |
 
-#### Record Accessor Enabled
+### Record Accessor Enabled
 
-This plugin enables the [Record Accessor](../../administration/configuring-fluent-bit/classic-mode/record-accessor.md) feature to specify the KEY. Using the _record accessor_ is suggested if you want to match values against nested values.
+Enable the [Record Accessor](../../administration/configuring-fluent-bit/classic-mode/record-accessor.md)
+feature to specify the KEY. Use the record accesso to match values against nested
```
Comment (alexakreizinger) [inline pipeline/filters/grep.md:23]:
```suggestion
feature to specify the KEY. Use the record accessor to match values against nested
```

File: pipeline/filters/grep.md [hunk 2]
```diff
@@ -35,20 +40,25 @@ In order to start filtering records, you can run the filter from the command lin
 {"log": "ggg"}
 ```
 
-### Command Line
+### Command line
 
-> Note: using the command line mode need special attention to quote the regular expressions properly. It's suggested to use a configuration file.
+When using the command line, pay special attention to quote the regular expressions.
```
Comment (alexakreizinger) [inline pipeline/filters/grep.md:45]:
```suggestion
When using the command line, pay close attention to quote the regular expressions.
```

File: pipeline/filters/grep.md [hunk 3]
```diff
@@ -35,20 +40,25 @@ In order to start filtering records, you can run the filter from the command lin
...
+Using a configuration file might be easier.
 
-The following command will load the _tail_ plugin and read the content of `lines.txt` file. Then the _grep_ filter will apply a regular expression rule over the _log_ field \(created by tail plugin\) and only _pass_ the records which field value starts with _aa_:
+The following command loads the [tail](../../pipeline/inputs/tail) plugin and
+reads the content of `lines.txt` file. Then the `grep` filter applies a regular
```
Comment (alexakreizinger) [inline pipeline/filters/grep.md:49]:
```suggestion
reads the content of `lines.txt`. Then the `grep` filter applies a regular
```
(or "the `lines.txt` file"?)

File: pipeline/filters/grep.md [hunk 4]
```diff
@@ -87,14 +99,21 @@ pipeline:
           match: '*'
 
 ```
+
 {% endtab %}
 {% endtabs %}
 
-The filter allows to use multiple rules which are applied in order, you can have many _Regex_ and _Exclude_ entries as required.
+The filter allows you to use multiple rules which are applied in order. You can
```
Comment (alexakreizinger) [inline pipeline/filters/grep.md:106]:
```suggestion
The filter lets you use multiple rules which are applied in order. You can
```

**Review by alexakreizinger — APPROVED** [review]
small suggestions but LGTM!

File: pipeline/filters/grep.md (hunk 2 above)
Comment (esmerel) [reply pipeline/filters/grep.md:45]: i thought i got that!.

**Pull request merged**